eng/common/scripts/Verify-ChangeLog.ps1

```diff
@@ -25,7 +25,13 @@ else
   Import-Module (Join-Path $PSScriptRoot modules Package-Properties.psm1)
   if ([System.String]::IsNullOrEmpty($Language))
   {
-    $Language = $RepoName.Substring($RepoName.LastIndexOf('-') + 1)
+    switch -Regex ($RepoName)
```

Comment:
If we are going to use a regex I think this should work. 

```
if ($RepoName -match "azure-sdk-for-(?<lang>[^-]*)") 
{
  $Language = $matches["lang"]
}
```

Reply:
By the way one of the improvements I've been thinking about is to setup a common ps1 file that we can have in each of the language repo's (outside of the eng\common directory) that can define things like `$global::Language = "<lang>"` That will allow us to parameterize variables and provide language specific methods directly in the language repo but use them in our common scripts so we don't always have to parse or pass them to all our functions.

Reply:
I created an issue to track that https://github.com/Azure/azure-sdk/issues/1544